src/drivers/St7789.cpp

```diff
@@ -62,7 +65,12 @@ void St7789::ColMod() {
 
 void St7789::MemoryDataAccessControl() {
   WriteCommand(static_cast<uint8_t>(Commands::MemoryDataAccessControl));
+// P8B Mirrored version sets MX bit high in MADCTL register to flip about vertical axis.
+#ifdef DRIVER_DISPLAY_MIRROR
+  WriteData(0x40); // MY(0) MX(1) MV(0) ML(0) RGB(0) 0 0 0
+#else
```

Comment:
This is better represented in a binary format. I'd prefer the comments to be in a similar format as in Cst816s.cpp
```suggestion
// P8B Mirrored version sets MX bit high in MADCTL register to flip about vertical axis.
#ifdef DRIVER_DISPLAY_MIRROR
  // [6] = Mirror columns(?)
  // etc.
  WriteData(0b01000000);
#else
```

Reply:
Okay, I adjusted the code.